garrysmod/gamemodes/terrortown/gamemode/gamemsg.lua

```diff
@@ -245,9 +249,13 @@ local function MuteTeam(ply, cmd, args)
 
    local t = tonumber(args[1])
    ply.mute_team = t
-
-   local name = (t != 0) and team.GetName(t) or "None"
-   ply:ChatPrint(name .. " muted.")
+   if t != MUTE_NONE and t != MUTE_ALL then
+   ply:ChatPrint(team.GetName(t) .. " muted.")
+   elseif t == MUTE_ALL then
+   ply:ChatPrint("All muted.")
+   else
+   ply:ChatPrint("None muted.")
+   end

```

Comment:
This needs indentation. Also couldn't team id <-> mute enum collision happen? I don't know what the team ids are in ttt so I don't know.